pkg/configv2/default.go

```diff
@@ -0,0 +1,165 @@
+package configv2
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/bacalhau-project/bacalhau/pkg/configv2/types"
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+const (
+	Second = types.Duration(time.Second)
+	Minuet = types.Duration(time.Minute)
+	Day    = types.Duration(time.Hour * 24)
+)
+
+// Default is the default configuration for a bacalhau node.
+var Default = types.Bacalhau{
+	API: types.API{
+		Address: "0.0.0.0:1234",
+	},
+	NameProvider: "puuid",
+	DataDir:      getDefaultDataDir(),
+	Orchestrator: types.Orchestrator{
+		Enabled: true,
+		Listen:  "0.0.0.0:4222",
+		NodeManager: types.NodeManager{
+			GCThreshold:       Day,
+			GCInterval:        10 * Minuet,
+			DisconnectTimeout: Minuet,
+		},
+		Scheduler: types.Scheduler{
+			WorkerCount:          runtime.NumCPU(),
+			HousekeepingInterval: 30 * Second,
+			HousekeepingTimeout:  2 * Minuet,
+		},
+		EvaluationBroker: types.EvaluationBroker{
+			VisibilityTimeout: Minuet,
+			MaxRetryCount:     10,
+		},
+	},
+	Compute: types.Compute{
+		Enabled: true,
+		Heartbeat: types.Heartbeat{
+			InfoUpdateInterval:     Minuet,
+			ResourceUpdateInterval: 30 * Second,
+			Interval:               15 * Second,
+		},
+		TotalCapacity: types.Resource{
+			CPU:    "1",
+			Memory: "1Gb",
+			Disk:   "1Gb",
+			GPU:    "0",
+		},
+		AllocatedCapacity: types.ResourceScaler{
+			CPU:    "80%",
+			Memory: "80%",
+			Disk:   "80%",
+			GPU:    "100%",
+		},
```

Comment:
Lets just keep AllocatedCapacity and make it accept both percentages and fixed values. We can keep the percentage implementation for now and introduce the fixed values in the future, but this means the values we accept today must end with "%"

Reply:
Will remove the `TotalCapacity` config and leave `AllocatedCapacity` unchanged.